lib/tasks.js

```diff
@@ -271,6 +271,19 @@ module.exports = async options => {
       await run(`!rm -rf ${dist.stageDir}`);
     }
 
+    if (github.release) {
+      getSpinner().stopAndPersist({
```

Comment:
My bad, but please use the `log` helper for this.

Reply:
Okay.